nf-test.config

```diff
@@ -6,4 +6,8 @@ config {
     profile "docker"
     withTrace false
 
+    stage {
```

Comment:
Yup, you're right. I think I instinctively did the same thing I did for snvphyl, but snvphyl is structured as a workflow (rather than a pipeline), so it made more sense there.

6fe181fb8b7aeb8b083ca2b5e9ad04f8dd0823c4

Reply:
Oh, also, I needed to specifically add the parameters for memory, since the pipeline test seemed to ignore the config file with parameters for testing.

Reply:
Thanks so much. This looks great 👍